src/olympia/amo/tests/test_sitemap.py

```diff
@@ -66,6 +70,43 @@ def test_amo_sitemap():
         assert sitemap.location(item) == reverse(item)
 
 
+def test_categories_sitemap():
+    sitemap = CategoriesSitemap()
+    # without any addons we should still generate a url for each category
+    empty_cats = list(sitemap.items())
+    assert empty_cats == [
+        *(
+            (category, 1)
+            for category in CATEGORIES[amo.FIREFOX.id][amo.ADDON_EXTENSION].values()
+        ),
+        *(
+            (category, 1)
+            for category in CATEGORIES[amo.FIREFOX.id][amo.ADDON_STATICTHEME].values()
+        ),
+    ]
+    # add some addons and check we generate extra pages when frontend would paginate
+    bookmarks_category = CATEGORIES[amo.FIREFOX.id][amo.ADDON_EXTENSION]['bookmarks']
+    shopping_category = CATEGORIES[amo.FIREFOX.id][amo.ADDON_EXTENSION]['shopping']
+    AddonCategory.objects.create(
+        addon=addon_factory(category=bookmarks_category), category=shopping_category
+    )
+    AddonCategory.objects.create(
+        addon=addon_factory(category=shopping_category), category=bookmarks_category
+    )
+    addon_factory(category=bookmarks_category)
+    addon_factory(category=shopping_category, status=amo.STATUS_NOMINATED)
+    # should be 3 addons in shopping (one not public, so 2 public), and 3 in bookmarks
+
+    patched_drf_setting = dict(settings.REST_FRAMEWORK)
+    patched_drf_setting['PAGE_SIZE'] = 2
+    with override_settings(REST_FRAMEWORK=patched_drf_setting):
+        cats_with_addons = list(sitemap.items())
+    # only one extra url, for a second bookmarks category page, because PAGE_SIZE = 2
+    extra = (bookmarks_category, 2)
+    assert extra in cats_with_addons
+    assert set(cats_with_addons) - set(empty_cats) == {extra}
```

Comment:
we should probably have some data for Android categories, no? https://addons.mozilla.org/en-US/android/extensions/category/experimental/

Reply:
#16910 for android - tbh I'm still not 100% on which urls should be included and which should be alternates (I see https://github.com/mozilla/addons-frontend/issues/6644 mentions android urls too).  For categories specifically, we do have category data, but we don't expose category pages on frontend so it's unclear (discussion in #16910 I guess).